.github/workflows/test.yml

```diff
@@ -37,8 +37,8 @@ jobs:
       - name: Run unit tests
         run: make test
 
-      - name: Run helper tests
-        run: make test-docker
+      - name: Run helper tests and sonar analysis for .NET
```

Comment:
Comment shouldn't be specific to .NET since this token will be used for all languages.

Reply:
Changed the comment to Run helper tests and sonar analysis